src/locale/locales/fr/messages.po

```diff
@@ -2059,7 +1954,7 @@ msgstr "Quitter Bluesky"
 
 #: src/screens/Deactivated.tsx:128
 msgid "left to go."
-msgstr ""
+msgstr "avant la fin."
```

Comment:
Here, the context matters: it's talking about how many users there is in the queue at that time.

```suggestion
msgstr "devant vous dans la file."
```